cmd/deploy/validate.go

```diff
@@ -15,9 +15,16 @@ package deploy
 
 import (
 	"github.com/okteto/okteto/pkg/env"
+	"github.com/okteto/okteto/pkg/validator"
 )
 
+// validateAndSet returns error when variables dont have expected format NAME=VALUE or NAME is not allowed
+// when variable is valid, it sets its value as env variable
 func validateAndSet(variables []string, setEnv func(key, value string) error) error {
+	if err := validator.CheckReservedVariablesNameOption(variables); err != nil {
```

Comment:
How it should behave if I add a reserved variable and run the command on remote? Maybe I want to deploy on a different namespace, What should be the best way to do it?

Reply:
validation is intended to be done when the cmd is starting, not getting any deeper in the code. when running the remote, as far as i know is using the deploy command as well with the flag remote?